src/components/expanded-state/AvailableNetworksv2.tsx

```diff
@@ -144,81 +138,91 @@ const AvailableNetworksv2 = ({
       actionKey: `${chain.id}`,
       actionTitle: useBackendNetworksStore.getState().getChainsLabel()[chain.id],
       icon: {
-        iconType: 'ASSET',
-        iconValue: `${useBackendNetworksStore.getState().getChainsName()[chain.id]}Badge${chain.id === ChainId.mainnet ? '' : 'NoShadow'}`,
+        iconType: 'REMOTE',
+        iconValue: {
+          uri: useBackendNetworksStore.getState().getChainsBadge()[chain.id],
+        },
       },
     }));
 
-  const MenuWrapper = availableChainIds.length > 1 ? ContextMenuButton : Box;
+  const Children = useMemo(() => {
```

Comment:
children is memo maybe it works I never tested returning components from useMemo